cloud/kubernetes/README.md

```diff
@@ -116,13 +116,37 @@ kubectl create -f cluster-init.yaml
 
 ### Secure mode
 
+#### Prerequisites
+
 **REQUIRED**: the kubernetes cluster must run with the certificate controller enabled.
 This is done by passing the `--cluster-signing-cert-file` and `--cluster-signing-key-file` flags.
 On minikube, you can tell it to use the minikube-generated CA by specifying:
 ```shell
 minikube start --extra-config=controller-manager.ClusterSigningCertFile="/var/lib/localkube/ca.crt" --extra-config=controller-manager.ClusterSigningKeyFile="/var/lib/localkube/ca.key"
 ```
 
+**REQUIRED**: because secure mode creates certificate signing requests and
```

Comment:
This paragraph is a bit rambly.

Reply:
Copied the text from my docs change, which was a bit less rambly. Look better?